source/extensions/tracers/xray/daemon.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
```

Comment:
To clarify, this was added as a means to serialize the messages to the daemon into JSON?

Perhaps a comment here to explain that?